### PR TITLE
Update expectations.md

### PR DESCRIPTION
**Pull request by Vira-V**

### CODE REVIEW

**Comment by OlyaM88** [comment]
leaving a comment as a part of the task)

**Comment by Feterberi** [comment]
May it come true! :-)

File: expectations.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 2. Отримати достатньо практичних навиків, щоб можна було себе впевнено почувати при проходженні співбесід на Trainee/Junior FrontEnd Developer.
 3. Зрозуміти і освоїти навики роботи з JavaScript на практиці.
```
Comment (IlonaShch) [inline expectations.md:4]: +1 комент

**Comment by IlonaShch** [comment]
+ ще один комент

**Review by dosandk — APPROVED** [review]
good 👍

**Pull request closed without merging**